tikv-client/src/main/java/com/pingcap/tikv/expression/AggregateFunction.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tikv.expression;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.Objects;
+
+public class AggregateFunction implements Expression {
+  public enum FunctionType {
+    Sum,
+    Count,
+    Min,
+    Max,
+    First
```

Comment:
What about `last`?

Reply:
MySQL does not have first/last but TiDB has a built-in firstrow for aggregate-less group by.